sdk/identity/azure-identity/src/main/java/com/azure/identity/DeviceCodeCredentialBuilder.java

```diff
@@ -82,10 +81,7 @@ public DeviceCodeCredentialBuilder disableAutomaticAuthentication() {
      * @return a {@link DeviceCodeCredential} with the current configurations.
      */
     public DeviceCodeCredential build() {
-        ValidationUtil.validate(getClass().getSimpleName(), new HashMap<String, Object>() {{
```

Comment:
These validations were focusing on the input being non-null.
Now the inputs have default values, so validation no longer needed.